src/main/java/codeu/model/data/Action.java

```diff
@@ -0,0 +1,26 @@
+package codeu.model.data;
+
+/** This a class of different actions that can be displayed as an activity. */
+public enum Action {
+  JOIN("RegisteringUser", true),
+  SEND("SendingMessage", true),
+  CREATE("CreatingConversation", true);
+
+  /** More info about the action taken */
+  private String content;
```

Comment:
You don't need the `content` field.  You're only using it for comparisons, but you should be doing comparisons directly against the enum value (see comments below).